src/components/EditorPage/Metrics/Metrics.tsx

```diff
@@ -26,20 +26,17 @@ export default class Metrics extends React.PureComponent<Props, State> {
   }
 
   componentWillReceiveProps(nextProps: Props) {
-    let isMetricExceeded = false
     for (let key in nextProps.metrics) {
       const metric = key as keyof SceneMetrics
       if (nextProps.metrics[metric] > nextProps.limits[metric]) {
-        if (!this.metricExceeded || this.metricExceeded !== metric) {
-          this.metricExceeded = metric
+        if (!this.metricsExceeded.includes(metric)) {
```

Comment:
I think that `includes` does not work with an array of objects, only values. Maybe we can use a `find` or `any`